src/nupic/algorithms/Cells4.cpp

```diff
@@ -30,7 +30,6 @@
 #include <sstream>
 #include <vector>
 
-#include "cycle_counter.hpp"
```

Comment:
We no longer need cycle_counter.hpp?  Cool.  We need to delete that from the dependencies.

Reply:
good spot!